src/main/java/az/edu/turing/business/FamilyService.java

```diff
@@ -51,9 +51,9 @@ public List<Family> countFamiliesWithMemberNumber(int numberOfPeople) {
                 .collect(Collectors.toList());
     }
 
-    public Family createNewFamily(Human father, Human mother) {
-        if (father == null || mother == null) throw new NullPointerException("father and mother cannot be null");
-        Family family = new Family(father, mother);
+    public Family createNewFamily(Human mother, Human father) {
+        if (mother == null || father == null) throw new NullPointerException("father and mother cannot be null");
```

Comment:
it is not good practice throw NullPointerException like this, instead you can create your custom Exception like BadRequestException